### ISSUE TITLE
Attempt to update a registry document is flagged as not changed when content has changed.

### ISSUE THREAD

**Issue by mparsons-ce** [CredentialEngine/CredentialRegistry#326]
The credential: "CyberSAFE (Securing Assets for End-Users) Exam CBS-210" has been set the status of deprecated. The current envelope/graph is: 
https://credentialengineregistry.org/ce-registry/envelopes/bd6c0223-311d-4bd2-9adb-9ef558950b1f
https://credentialengineregistry.org/graph/ce-d80e6f22-1cbb-4b8f-97ab-25b255274b92

The json from the following gist has several differences, including the new status and owner:
https://gist.github.com/mparsons-ce/bca7225457c075a06eddee0968cab0ba

When trying to publish the latter JSON document I get the not changed condition.
The latter is only supposed to occur if the payload being uploaded is the same as that in the current envelope. 

First, I would like to know why this is occurring. 
Second, regardless of the answer to the latter, it may be useful to have an option to bypass the process that compares the update with the current content and force an update.

**Comment by mparsons-ce** [comment]
@excelsior Any update on this? I tried this again today, and was not able to get a credential to update.

**Comment by excelsior** [comment]
@mparsons-ce 

I've tried updating this credential locally, and it worked as expected. It's not possible to publish the document on behalf of an organization, because it isn't associated with any organization. So I encoded the resources from the gist you posted using my own key and published the resulting envelope via the `POST /envelopes` endpoint. The resource was updated and the `changed` flag in the response was `true`.

Are you updating the credential some other way?

**Comment by mparsons-ce** [comment]
@excelsior This credential did have a transfer of owner. 
The latter may not be a logical reason for issues, but it is a different aspect from other credentials.

**Comment by excelsior** [comment]
Currently, this envelope doesn't belong to any organization. I checked its version history and it seems it never had one. Maybe I'm missing something, though. How exactly was the ownership transferred?

Could you elaborate on how the updates are performed? Are you doing it on behalf of an organization or just publishing a signed envelope? Please share a cURL snippet privately or something like that, if possible.

**Comment by mparsons-ce** [comment]
This credential was original published using the Credential Engine managed keys (not registry managed keys). On April 27, 2020 I did a transfer of owner to CertNexus (ce-c04dd8ac-d473-475f-8901-d47f201afe64). After April 27th, I have not been able to update the credential. 

I just redid the transfer of owner action. Does the credential have an owner now?

**Comment by mparsons-ce** [comment]
@excelsior After doing the second transfer of owner, I tried to republish the credential.
This time it worked??
I will have to check on other credentials that had a transfer of owner to determine if they have issues being republished.

**Comment by siuc-nate** [comment]
Mike will attempt to recreate the situation to test this.

**Comment by mparsons-ce** [comment]
I tested this on staging:

- published a credential with owner CE
- transfer ownership to Participate
- updated credential and published as participate
The credential was updated properly. 

The latter was all done via Postman. I did a similar test doing the publish from the editor. It too was successful. 
Closing this issue.

**Comment by mparsons-ce** [comment]
I have encountered this issue again
I had published the pathway:
https://sandbox.credentialengineregistry.org/graph/ce-c14ca5cd-1b75-4bf4-8321-3be1ab6a1256
I transfered the ownership 
from: ce-12ccd14d-3b90-4e36-bc1b-88c3231e2610 
to: ce-5ea303a9-0cba-42db-b828-38d2f1fb890d 

I then tried to republish the pathway. I added the text: ' CHANGED'.
I got the message indicating the data had not changed, so the envelope was not updated. 
I then updated the description and tried to republish - again got the message that the envelope was not updated. 

I tried transferring the owner back to the original and then again. Still the same issue.
